src/utils/sendEmail.js

```diff
@@ -0,0 +1,26 @@
+import nodemailer from "nodemailer"
+import dotenv from "dotenv";
+dotenv.config(); 
+ 
+const configEmail = async  ({ email, subject, content } = {}) => {
+    try {
+        const transporter = nodemailer.createTransport({
```

Comment:
great work yob! but we have different sendemail function it's better to use the same